README.md

```diff
@@ -35,10 +35,36 @@ following files:
   1. as final step, return the bare-metal machine to the CentOS CI for other
      users (it will be re-installed with a minimal CentOS environment again)
 
+- `e2e.groovy` is the Jenkins Pipeline responsible for running End-to-End tests on
+  a multi-node kubernetes cluster hosted on [Centos CI](centos_ci).
```

Comment:
Note that the format is `[title][id]` and the `[id]` needs to be available at the bottom of the document.